tests/integration/test_intelligence.py

```
@@ -591,3 +627,128 @@ def test_get_equity_offering_by_cik(self, fmp_client: FMPDataClient, vcr_instanc
+            articles = self._handle_rate_limit(
+                fmp_client.intelligence.get_stock_symbol_news,
+                "AAPL",
+                page=0,
+                limit=5,
+            )
+            assert isinstance(articles, list)
+            if articles:
+                assert isinstance(articles[0], StockNewsArticle)
+
+    def test_get_historical_social_sentiment(
+        self, fmp_client: FMPDataClient, vcr_instance
```

Ruff (0.14.13)

Unused method argument: `vcr_instance`

(ARG002)